### ISSUE TITLE
Remember to check issues on Other repos

### ISSUE THREAD

**Issue by adrlau** [Programvareverkstedet/Drift#87]
andre repoer som nettsiden har også egne issue boards. Noen ganger burde de også sjekkes, om det er viktig for PVV.

**Comment by adrlau** [comment]
Ja, jeg skjønner om det er fint å ha ting ett sted, men lagde dette issue for å minne oss på å se flere steder, til kodeting feks, om det er av relevans til infra.
Blir også litt clutter om småissues til kode, kommer her.
Kunne eventuelt hatt ett egent board for det, men er fortsatt litt rotete.

**Comment by felixalbrigtsen** [comment]
Synes samtidig ikke at dette her er et drift-issue. PVV er ikke først og fremst drift, det er alt mulig som har med programvare å gjøre. Det virker litt lukket å sette sånne ting som en driftsoppgave, og at vi heller bør prøve å være prinsippfaste og oppmuntre alle PVV-ere til å delta i kodeting, som på hackekveld. Det betyr også at _vi_ i drift kan og bør jobbe på det, men alle andre dager enn driftkveld imo.

**Comment by dali99** [comment]
Jeg skrev orginalt en relativt lang kommentar her om hvordan jeg syntest det er vanskelig å definere drift på noe annen måte enn "hacking med formål om å forbedre PVV-prosjekter". Men jeg kom til konklusjonen om at denne diskusjonen bør tas i et hvilken som helst annet medium enn github-kommentarer.

Jeg foreslår at vi tar en ordentlig diskusjon om dette på neste driftskveld.

Jeg tror jeg heller mot å være enig med Adrian med at å gjøre triage og sync også på andre repoer kan være et nyttig punkt å ha på driftsmøteagendaen. Men jeg er også enig med felix at det er fint om vi kan overbevise en annen gruppe som kanskje ikke vil gjøre ops til å være med på disse tingene.
